local-test-nx-playwright.sh

```diff
@@ -1,4 +1,7 @@
-LOCAL_TEST_WORKSPACE="$(pwd)/../onyx-nx"
+if [[ -z "${LOCAL_TEST_WORKSPACE}" ]]; then
```

Comment:
The script will fail if the user will provide a relative path instead of an absolute path.
We need to find a better strategy here.

Reply:
I've tested this, and it does actually work when a relative path is provided.